tajo-core/src/main/java/org/apache/tajo/engine/planner/global/GlobalPlanner.java

```diff
@@ -991,7 +992,7 @@ public LogicalNode visitProjection(GlobalPlanContext context, LogicalPlan plan,
         for (DataChannel dataChannel : masterPlan.getIncomingChannels(execBlock.getId())) {
           // This data channel will be stored in staging directory, but RawFile, default file type, does not support
           // distributed file system. It needs to change the file format for distributed file system.
-          dataChannel.setStoreType("TEXT");
+          dataChannel.setStoreType(BuiltinStorages.DRAW);
```

Comment:
How about adding a final variable for the default store type and using it in the class?